plugins/ufm_consumer_plugin/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>UI_hello_word_plugin</name>
```

Comment:
why do we need such file? 
and if we need it, we should change the name instead of the UI hello world